addons/html_editor/static/src/fields/property_value.js

```diff
@@ -68,7 +68,7 @@ patch(PropertyValue.prototype, {
         }
 
         return {
-            content: this.propertyValue,
+            content: markup(this.propertyValue),
```

Comment:
markup should be moved up in the chain where the safe value is actually produced